src/dissect/dissect.c

```diff
@@ -127,6 +134,12 @@ static int parse_argv(int argc, char *argv[]) {
                         break;
                 }
 
+                case ARG_VERITY_DATA:
+                        r = parse_path_argument_and_warn(optarg, true, &arg_verity_data);
```

Comment:
i don't think the second argument should be true.

It should only be used when parsing chroot paths. But here this doesn't apply

Reply:
Ok, I misunderstood what the flag was for, fixed